EncryptionTests/Encryption Tests/DatastoreEncryptionTests.m

```diff
@@ -192,4 +193,15 @@ - (void)testInitReturnsNilIfEncryptionKeyProviderDoesNotReturnTheKeyUsedToOpenTh
                  @"DB can not be opened with a wrong key so the datastore can not be initialised");
 }
 
+- (void)testTextSearchIsAvailable
+{
+    CDTHelperFixedKeyProvider *provider = [CDTHelperFixedKeyProvider provider];
+    
+    CDTDatastore *datastore = [self.factory datastoreNamed:@"test_copyprovider"
+                                 withEncryptionKeyProvider:provider
+                                                     error:nil];
+    
+    XCTAssertTrue([datastore isTextSearchEnabled], @"It should be activated");
```

Comment:
Pretty sure this test is redundant with the one above.  `CDTDatastore+Query -isTextSearchEnabled` ultimately rests upon the same bit of code: https://github.com/cloudant/CDTDatastore/blob/master/Classes/common/query/CDTQIndexManager.m#L354-L372.